sql/db.go

```diff
@@ -0,0 +1,123 @@
+package sql
+
+import (
+	"database/sql"
+	"errors"
+	"time"
+
+	"github.com/Masterminds/squirrel"
+	bt "github.com/elh/bitempura"
+)
+
+var _ DB = (*TableDB)(nil)
+
+// DB is a SQL-backed, SQL-queryable, bitemporal database.
```

Comment:
- add FIXME about how DB and NewTableDB are unfinished
- add FIXME to package comment
- add WARN that functions are unimplemented